### ISSUE TITLE
Update Iso

### ISSUE THREAD

**Issue by idnovic** [v-fox/live_opensuse_hsf#22]
Please upload a new iso. 
I am not able to build an iso myself following your instructions. Because I am presented with an error message I was not able to solve. Error was about kiwi boot arguments only supported by oem or pxe profile but not iso mode. 

Thank you!

**Comment by v-fox** [comment]
Yeah, I need to make a tested release and build it. I've been postponing it for ridiculous amount of time. Some changes in openSUSE build infrastructure were lowering my enthusiasm for a while. Such as kiwi changes, denial of my feature request for it to auto-create Windows-usable exfat partition and obsolescence of their universal installer that worked for any custom builds.

What exact parameter that was ? Because earlier it worked fine on older version of kiwi.

**Comment by idnovic** [comment]
I copied the following from on of your files:

```
<type image="iso" primary="true" checkprebuilt="false"
		boot="isoboot/suse-tumbleweed" firmware="uefi" bootprofile="custom" bootkernel="custom"
		flags="overlay" hybrid="true" hybridpersistent="true" hybridpersistent_filesystem="ext4"
```

As far as I can recall I had the following errors:

image=iso does not work with boot arguments. 
hybridpersistent needed to be deleted because it did not build with it. And I needed to add a filesystem=ext4. But I was presented with more errors and gave up.

**Comment by v-fox** [comment]
I see. `iso` mode is required fir a Live build and [`hybrid`](https://osinside.github.io/kiwi/building_images/build_live_iso.html) is needed for when you put the image on a USB flash drive. It's also for use with TumbleWeed only because it depends on my repoes that build for TW only. Although, technically, build host should not matter for kiwi's build chroot but maybe it does. But it's also essential to force it to use [my kernel package](https://build.opensuse.org/package/show/home:X0F:HSF:Kernel/kernel-HSF) instead of vanilla, earlier that was required parameters to do so.

I will look into it later.
